requirements.txt

```diff
@@ -4,7 +4,7 @@ argcomplete
 colorama
 hid
 hjson
-jsonschema>=3
+jsonschema>=4
```

Comment:
Will also require <https://github.com/qmk/qmk_cli/blob/master/setup.cfg#L45> to be aligned.

Reply:
Correct. Opened https://github.com/qmk/qmk_cli/pull/119.